tests/foreman/cli/test_host.py

```diff
@@ -2323,6 +2323,39 @@ def test_syspurpose_end_to_end(
             }
         )
 
+@pytest.mark.rhel_ver_match('[^7]')
+def test_multi_cv_reg_not_allowed(
+    module_org,
+    module_activation_key,
+    module_lce,
+    module_lce_library,
+    module_published_cv,
+    module_promoted_cv,
+    target_sat,
+    rhel_contenthost,
+):
+    """Attempt to register a host to multiple content view environments.
+
+    :expectedresults: allow_multiple_content_views setting is not exposed, and defaults to false.
+    So registration fails because multiple environments are not allowed.
+
+    :CaseImportance: Critical
+
+    :parametrized: yes
+    """
+
+    # Register with global reg, just to get the sub-man config and certs right
+    rhel_contenthost.register(module_org, None, module_activation_key.name, target_sat)
+
```

Comment:
Lets add some extra assertions here to make sure the inital registration succeeds(helps for debugging tests if it ever fails) 
something like: 

```assert result.status == 0```
```assert rhel_contenthost.subscribed```